packages/combo-box/test/filtering.test.js

```diff
@@ -268,29 +315,21 @@ describe('filtering items', () => {
     it('should remove focus while loading', () => {
       setInputValue('foo');
       comboBox.filteredItems = ['foo'];
-      expect(comboBox._focusedIndex).to.eql(0);
+      expect(comboBox.opened).to.be.true;
+      expect(getFocusedItemIndex()).to.equal(0);
 
       comboBox.loading = true;
-
-      expect(comboBox._focusedIndex).to.eql(-1);
+      expect(getFocusedItemIndex()).to.equal(-1);
     });
 
     it('should focus on filtered value', () => {
       comboBox.filteredItems = ['foo'];
       setInputValue('bar');
-      expect(comboBox._focusedIndex).to.eql(-1);
+      expect(getFocusedItemIndex()).to.equal(-1);
 
       comboBox.filteredItems = ['foo', 'bar', 'baz'];
 
-      expect(comboBox._focusedIndex).to.eql(1);
-    });
-
-    it('should focus on value when filter is empty', () => {
```

Comment:
**note:** This test has been dropped in favor of: 

https://github.com/vaadin/web-components/pull/3600/files#diff-7fa09446be562a2e3920fce61ff91e9083fe0e11ea590a43a5e9dc9107f4da6dR451-R454.